lib/types.go

```diff
@@ -64,10 +64,18 @@ func (e APIError) Error() string {
 
 // AccountInfo represents account's info.
 type AccountInfo struct {
-	Address  string `json:"address"`
-	PubKey   string `json:"pubkey"`
+	// Address is the wallet address
+	Address string `json:"address"`
```

Comment:
We keep it `Address` and `Pubkey` for backward compatibility? Do we want to change it in the future? If so, I would duplicate `Address` and `PubKey` and add `WalletAddress` and `WalletPubKey`. Then, status-react can upgrade in any moment.

Reply:
Also, in `SignupResponse` we changed the fields and break compatibility so it's a bit inconsistent.

Reply:
thank you @adambabik , I added both address and walletAddress so it's more clear